### PR TITLE
Hello Microverse: Add initial files and set up the Linters

### PR DESCRIPTION
**Pull request by rmiyoyo**
## Things Worked On
- Added index.html, style.css, README.md, and LICENSE files.
- Configured linters for the project.
- Fixed indentation issues in css.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<a name="readme-top"></a>
+
```
Comment (OyePriscilla) [inline README.md:2]: - [x] Awesome adding README file, but for it to be more informative and engaging; kindly ensure that all the sections of [Microverse template](https://github.com/microverseinc/readme-template) are customized and filled out. The only optional section is FAQ and you can indicate "Not yet available" for the live demo section. Kindly don't forget to update your table of contents and ensure that no link(s) is/are broken. ℹ️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @rmiyoyo 👏 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Add .html and .css files 👍 
Observe GitHub Flow ✔️
Linters check passed 👍  


## Required Changes ♻️

- [x] Kindly make your  branch name to be more descriptive. The branch name should be a summary of what you did in the branch. **Kindly checkout to the new branch and use the correct branch name and mention the old PR in the new PR.**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rmiyoyo) [reply README.md:2]: Hello @OyePriscilla, I have made the changes. Should I create a new pull request?

Comment (rmiyoyo) [reply README.md:2]: I have created a new [pull request](https://github.com/rmiyoyo/Hello-Microverse/pull/2) Thanks @OyePriscilla

Comment (OyePriscilla) [reply README.md:2]: @rmiyoyo That's awesome, keep doing the good work!

**Pull request merged**